src/metabase/upload.clj

```diff
@@ -224,7 +219,7 @@
     (contains? ys x) x
     :else            (recur xs ys)))
 
-(defn- lowest-common-ancestor [type-a type-b]
+(defn- most-specific-common-ancestor [type-a type-b]
```

Comment:
I don't mind this name change, but it's now inconsistent with `lowest-common-member` and `first-common-ancestor` which both appear in this namespace but refer to the same idea. Ideally you should use the same name to refer to the same idea